src/braket/circuits/gates.py

```diff
@@ -1514,6 +1514,13 @@ def adjoint(self) -> List[Gate]:
     def _to_jaqcd(self, target: QubitSet):
         return ir.ECR.construct(targets=[target[0], target[1]])
 
+    def _to_openqasm(
+        self, target: QubitSet, serialization_properties: OpenQASMSerializationProperties, **kwargs
+    ):
+        target_qubit0 = serialization_properties.format_target(int(target[0]))
+        target_qubit1 = serialization_properties.format_target(int(target[1]))
+        return f"ecr {target_qubit0}, {target_qubit1};"
```

Comment:
nit: I would add a `_` before the index
```suggestion
        return f"ecr {target_qubit_0}, {target_qubit_1};"
```